packages/react-migration-v8-v9/src/stories/ThemeShim/createV8Theme/FluentComponentSamples.tsx

```diff
@@ -0,0 +1,208 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
This file doesn't need much consideration. It's really just copied from the v8 demo and meant to give a visual confirmation of the user's theme.